src/components/RadioFilter/RadioFilters.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { COURSE_TYPES } from '../../constants/course-data.js';
+import './RadioFilters.css';
+
+
+function RadioFilters({ courseType, setCourseType, filterCoursesAfterRadioFilter }) {
+  function handleClick(e) {
+    setCourseType(e.target.value);
+    filterCoursesAfterRadioFilter(e.target.value);
+  }
+
+  const courseTypeRadioInputs = [];
+
+  COURSE_TYPES.forEach(type => {
```

Comment:
I didn't see you render courses via Array.map. feel free to refactor it to use same method